pkg/cluster/delete.go

```diff
@@ -484,13 +531,17 @@ func (m *manager) Delete(ctx context.Context) error {
 	if err != nil {
 		return err
 	}
-
 	if m.doc.OpenShiftCluster.UsesWorkloadIdentity() {
 		m.log.Printf("deleting cluster MSI certificate")
 		err = m.deleteClusterMsiCertificate(ctx)
 		if err != nil {
 			return err
 		}
+
+		err = m.deleteFederatedCredentials(ctx)
```

Comment:
Nit: we can call this "deleteIdentityFederations"

Reply:
updated